docs/new-architecture-library-intro.md

```diff
@@ -562,6 +562,28 @@ RCT_EXPORT_METHOD(moveToRegion:(nonnull NSNumber *)reactTag
 
 **Android**
```

Comment:
Since you're at this, could you udpate line 548:

```diff
-### Updating Native implementation [iOS]
+### Updating Native implementation
```

Reply:
Will do!